### DISCUSSION TITLE
posts/Jekyll-to-Hexo/

### DISCUSSION THREAD

**Discussion by giscus** [JHTNT/jhtnt.github.io#1] [Announcements]
# posts/Jekyll-to-Hexo/

前言 原本 Jekyll 使用的是 Layon 主題，想著一開始用極簡主題就好，有需要什麼功能再額外加上去。結果東加西加，網站的程式碼變得很雜亂，不如直接換一個主題。 後來想起之前看過朋友用 Hexo 架的網站看起來滿漂亮的，可以調整的設定不少，也支援 Markdown。因此打算從 Jekyll 轉

https://jhtnt.github.io/posts/Jekyll-to-Hexo/

**Comment by JHTNT** [comment]
留言測試

**Reply by JHTNT** [reply]
回覆測試
